classes/Twig/UpdateSteps.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace PrestaShop\Module\AutoUpgrade\Twig;
+
+use PrestaShop\Module\AutoUpgrade\UpgradeTools\Translator;
+
+class UpdateSteps
+{
+    const VERSION_CHOICE = 'version-choice';
+    const UPDATE_OPTIONS = 'update-options';
+    const BACKUP = 'backup';
+    const UPDATE = 'update';
+    const POST_UPDATE = 'post-update';
```

Comment:
To restrict the possible values that can be sent to the method, I advise to prefix your constants with a same word. For instance:

```suggestion
    const STEP_VERSION_CHOICE = 'version-choice';
    const STEP_UPDATE_OPTIONS = 'update-options';
    const STEP_BACKUP = 'backup';
    const STEP_UPDATE = 'update';
    const STEP_POST_UPDATE = 'post-update';
```